homeassistant/components/zamg/config_flow.py

```diff
@@ -92,12 +91,6 @@ async def async_step_import(self, config: dict[str, Any]) -> FlowResult:
             translation_key="deprecated_yaml",
         )
 
-        for entry in self.hass.config_entries.async_entries(DOMAIN):
-            if station_id in entry.data[CONF_STATION_ID]:
-                return self.async_abort(
-                    reason="already_configured",
-                )
-
         if self._client is None:
             self._client = ZamgData()
             self._client.session = async_get_clientsession(self.hass)
```

Comment:
Can the api call on line 98 raise an error?

Reply:
No, it returns a list or None

Reply:
If it returns `None` we can't do a membership test as we do on line 98 now.